openslides_backend/action/relations/single_relation_handler.py

```diff
@@ -40,18 +40,27 @@
 )
 from ...shared.typing import DeletedModel, ModelMap
 
-ListType = Union[List[int], List[str], List[FullQualifiedId]]
+Identifier = Union[int, str, FullQualifiedId]
```

Comment:
Please check if these type definitions should really live here or somewhere else. I am not sure whether all of the belong to this special single relation case.